index.html

```diff
@@ -1,155 +1,246 @@
             </div>
-        </section>
+          </div>
+          <div class="work">
+            <div class="project-info">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>JavScript</li>
+                <li>html</li>
+              </ul>
+              <button>See Project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="about-section">
+        <h2>About me</h2>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don't hestiate to contact me.
+        </p>
+        <button>Get My Resume</button>
+      </section>
+      <section class="skills-section">
+        <div class="skill-box language">
+          <img src="img/icon-languages.png" alt="Language" />
+          <h2>Language</h2>
+          <ul>
+            <li>JavaScript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>CSS</li>
+          </ul>
+        </div>
+        <div class="skill-box framework">
+          <img src="img/icon-frameworks.png" alt="Framework" />
+          <h2>Frameworks</h2>
+          <ul>
+            <li>Bootstrap</li>
+            <li>Ruby on rails</li>
+            <li>RSpec</li>
+            <li>Capybara</li>
+            <li>Selenium</li>
+          </ul>
+        </div>
+        <div class="skill-box skills">
+          <img src="img/icon-skills.png" alt="Skills" />
+          <h2>Skills</h2>
+          <ul>
+            <li>Codekit</li>
+            <li>GitHub</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
+      </section>
+      <section class="contact-section">
+        <div class="contact">
+          <div class="text-contact">
+            <h3>
+              I'm always interested in hearing about new projects, so if you'd
+              like to chat please get in touch.
+            </h3>
+          </div>
+          <div class="contact-form">
+            <form action="#" class="contact-form">
+              <input
+                type="text"
+                name="full-name"
+                id="name"
+                placeholder="Full name"
+                maxlength="30"
+                required
+              />
+              <input
+                type="email"
+                name="email"
+                id="email"
+                placeholder="Email Address"
+                maxlength="30"
+                required
+              />
+              <textarea
+                name="textarea"
+                id="textarea"
+                cols="35"
+                rows="8"
+                placeholder="Enter the Text Here"
+                maxlength="500"
+                required
+              ></textarea>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please give the correct style to all your `inputs`, I think you forgot to give the following styles: 
- 1. When the user starts typing in the inputs, the `text` color should be black. 
- 2. When the user clicks on Inputs, the color of the input `border`s should be green(`#60C095`) as shown in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988&t=2SxPJKlNwtIkjtSA-0). 
- 3. Please note the distance between the `inputs` and their `size`, they are not the same as the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=2SxPJKlNwtIkjtSA-0).

![image](https://user-images.githubusercontent.com/63915024/212883165-3df85e3e-fee3-4de7-95f6-27d7827af5f5.png)